cxx-qt-gen/src/parser/mod.rs

```diff
@@ -0,0 +1,169 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+pub mod qobject;
+
+use qobject::{find_and_merge_cxx_qt_item, find_qobjects, ParsedCxxQtData};
+use syn::{token::Brace, ItemMod, Result};
+
+/// A struct representing a module block with CXX-Qt relevant syn::Item's
+/// parsed into ParsedCxxQtData, to be used later to generate Rust & C++ code.
+///
+/// syn::Item's that are not handled specially by CXX-Qt are passed through for CXX to process.
```

Comment:
```suggestion
/// A struct representing a module block with CXX-Qt relevant [syn::Item]'s
/// parsed into ParsedCxxQtData, to be used later to generate Rust & C++ code.
///
/// [syn::Item]'s that are not handled specially by CXX-Qt are passed through for CXX to process.
```

Rustdoc is smart enough to link keywords from other crates :)

Reply:
Done